hobbify/habits/serializers/recurrences.py

```diff
@@ -0,0 +1,35 @@
+""" Recurrences serializers """
+
+# Django REST Framework
+from rest_framework import serializers
+
+# Models
+from hobbify.habits.models import Habit
+from hobbify.users.models import User
+from hobbify.habits.models import RecurrentInstance
+
+# Serializers
+from hobbify.users.serializers import UserModelSerializer
+from hobbify.habits.serializers import HabitModelSerializer
+
+# Dateutil
+from dateutil.rrule import rrule, DAILY, MONTHLY, WEEKLY
```

Comment:
I don't think you use any of this in your code. This is easily solved by automating code style and static analysis.